java/src/jmri/jmrit/logix/WarrantPreferencesPanel.java

```diff
@@ -261,7 +313,7 @@ private JPanel speedRosterPanel(boolean vertical) {
                 b3.setSelected(true);
                 break;
             default:
-                // fall out
+                // fall through
```

Comment:
Nothing is falling through here, and I do not think falling through is desired here, so this comment needs to be reverted to avoid later confusion.

The ```break;``` just before the ```default:``` case ensures the prior case does not fall through in the switch (generally it is safer not to fall through in a switch statement).

Reply:
Reverted